Source/WebGPU/WebGPU/Texture.mm

```diff
@@ -2157,7 +2157,8 @@ static bool textureViewFormatCompatible(WGPUTextureFormat format1, WGPUTextureFo
         return 0;
     }
     NSUInteger blocksInWidth = textureWidth / blockWidth;
-    return Texture::texelBlockSize(format) * blocksInWidth * sampleCount;
```

Comment:
likely this is not possible to overflow, since `texelBlockSize` and `texelBlockWidth` can be at most `16`. There is a risk someone passes `sampleCount` without validating it, but it is also an error to pass anything other than `1` or `4` for sample count.

Seems fine to leave the checked arithmetic however